rlbot_gui/bot_management/downloader.py

```diff
@@ -31,6 +50,12 @@ def download_and_extract_zip(download_url: str, local_folder_path: Path,
         with zipfile.ZipFile(downloaded_zip_path, 'r') as zip_ref:
             zip_ref.extractall(local_folder_path)
 
+        if not os.path.exists(os.path.join(local_folder_path, local_subfolder_name)):
+            folder_name = os.listdir(local_folder_path)[0]
+            os.rename(os.path.join(local_folder_path, folder_name), os.path.join(local_folder_path, local_subfolder_name))
+    return True
```

Comment:
What's this doing? Please document better in code, e.g. explain what local_subfolder_name is for.